tests/t530-operator.c

```diff
@@ -0,0 +1,191 @@
+/// @file
```

Comment:
This test checks the linearization of a linear qfunction. I test both the point values and the action of the linearization in the mass operator.